lib/serve/rest-api/src/api/routes.py

```diff
@@ -15,34 +15,22 @@
 """Model information routes."""
 
 import logging
-import os
 
-from fastapi import APIRouter, Depends
+from fastapi import APIRouter
 from fastapi.responses import JSONResponse
 
-from ..auth import OIDCHTTPBearer
 from .endpoints.v1 import embeddings, generation, models
 from .endpoints.v2 import litellm_passthrough
 
 logger = logging.getLogger(__name__)
 
 router = APIRouter()
 
-if os.getenv("USE_AUTH", "true").lower() == "false":
-    dependencies = []
-    logger.info("Auth disabled")
-else:
-    security = OIDCHTTPBearer()
-    dependencies = [Depends(security)]
-    logger.info("Auth enabled")
-
-
-router.include_router(models.router, prefix="/v1", tags=["models"], dependencies=dependencies, deprecated=True)
-router.include_router(embeddings.router, prefix="/v1", tags=["embeddings"], dependencies=dependencies, deprecated=True)
-router.include_router(generation.router, prefix="/v1", tags=["generation"], dependencies=dependencies, deprecated=True)
-router.include_router(
-    litellm_passthrough.router, prefix="/v2/serve", tags=["litellm_passthrough"], dependencies=dependencies
-)
+
+router.include_router(models.router, prefix="/v1", tags=["models"], deprecated=True)
+router.include_router(embeddings.router, prefix="/v1", tags=["embeddings"], deprecated=True)
+router.include_router(generation.router, prefix="/v1", tags=["generation"], deprecated=True)
```

Comment:
we've had these under the deprecation notice for a while now. this is already a large change, but maybe let's get rid of these and all associated code in a future commit if there's time. lets us remove more stuff from the readme.

Reply:
Maybe we can add that to the backlog for 3.0.1